publications.rkt

```diff
@@ -510,10 +487,606 @@
+                 "European Symposium on Programming (ESOP)"
+                 2014
+                 "http://dx.doi.org/10.1007/978-3-642-54833-8_3")
+    (publication "CFA2: a Context-Free Approach to Control-Flow Analysis"
+                 "Dimitrios Vardoulakis and Olin Shivers"
+                 ESOP
+                 2010
+                 "http://www.ccs.neu.edu/home/dimvar/papers/cfa2-NU-CCIS-10-01.pdf")
+    (publication "Modular rollback through control logging: a pair of twin functional pearls"
+                 "Olin Shivers and Aaron Joseph Turon"
+                 "International Conference on Functional Programming (ICFP)"
+                 2011
+                 "http://doi.acm.org/10.1145/2034773.2034783")
+    (publication "Pushdown flow analysis of first-class control"
+                 "Dimitrios Vardoulakis and Olin Shivers"
+                 "International Conference on Functional Programming (ICFP)"
+                 2011
+                 "http://www.ccs.neu.edu/home/dimvar/papers/cfa2-1st-class.pdf")
+    (publication "Ordering multiple continuations on the stack"
+                 "Dimitrios Vardoulakis and Olin Shivers"
+                 "PEPM"
+                 2011
+                 "http://www.ccs.neu.edu/home/dimvar/papers/rcps-NU-CCIS-11-01.pdf")
+    (publication "Bottom-up beta-reduction: Uplinks and lambda-DAGs"
+                 "Olin Shivers and Mitchell Wand"
+                 "Fundam. Inform"
+                 2010
+                 "http://dx.doi.org/10.3233/FI-2010-328")
+    (publication "CFA2: A Context-Free Approach to Control-Flow Analysis"
+                 "Dimitrios Vardoulakis and Olin Shivers"
+                 "European Symposium on Programming (ESOP)"
+                 2010
+                 "http://dx.doi.org/10.1007/978-3-642-11957-6_30")
+    (publication "Building language towers with Ziggurat"
+                 "David Fisher and Olin Shivers"
+                 "J. Funct. Program"
+                 2008
+                 "http://dx.doi.org/10.1017/S0956796808006928")
+    (publication "Exploiting reachability and cardinality in higher-order flow analysis"
+                 "Matthew Might and Olin Shivers"
+                 "J. Funct. Program"
+                 2008
+                 "http://dx.doi.org/10.1017/S0956796808006941")
+    (publication "Why teach programming languages"
+                 "Olin Shivers"
+                 "SIGPLAN Notices"
+                 2008
+                 "http://doi.acm.org/10.1145/1480828.1480856")
+    (publication "Trusted Theorem Proving: A Case Study in SLD-Resolution"
+                 "Konstantine Arkoudas and Olin Shivers"
+                 "ISoLA"
+                 2008
+                 "http://dx.doi.org/10.1007/978-3-540-88479-8_56")
+    (publication "Analyzing the environment structure of higher-order languages using frame strings"
+                 "Matthew Might and Olin Shivers"
+                 "Theor. Comput. Sci"
```

Comment:
Are abbreviations like these auto-generated?
(If not, I really think they should be `define`d)